ingest/rules/curate.smk

```diff
@@ -107,6 +107,8 @@ rule curate:
                 --abbr-authors-field {params.abbr_authors_field} \
             | augur curate apply-geolocation-rules \
                 --geolocation-rules {input.all_geolocation_rules} \
+            |  jq -c --arg GENBANK "{params.id_field}" \
```

Comment:
If a workflow used `strain` as the `id_field`, this would create invalid URLs...Hmm, might be best to add a new config param (e.g. `genbank_accession_field`) that is explicitly for this command.

Reply:
Ah, good point, especially if we use `id_field=strain` for segmented viruses (e.g. flu? I assume in that case strain name is in the header line of sequences.fasta instead of genbank) to match isolates across tangle trees.

I can add an explicit `genbank_accession_field` here

Reply:
Thanks, added an explicit `config['curate']['genbank_accession']` field during b2936f8097376a227a19a4099842129c468ba38c